giraffe/src/components/geo/CircleMarkerLayer.tsx

```diff
@@ -0,0 +1,125 @@
+// Libraries
+import React, {createRef, FunctionComponent, useLayoutEffect} from 'react'
+import {CircleMarker} from 'react-leaflet'
+
+// Utils
+import {
+  calculateMinAndMax,
+  getColor,
+  normalizeValue,
+} from './dimensionCalculations'
+import {defineToolTipEffect, formatValue} from './processing/toolTips'
+
+// Types
+import {GeoTable} from './processing/GeoTable'
+import {GeoTooltip} from './GeoTooltip'
+import {Config} from '../../types'
+import {GeoCircleViewLayer} from '../../types/geo'
+
+const DEFAULT_RADIUS = 50
+
+interface Props {
+  radiusFieldName: string
+  colorFieldName: string
+  table: GeoTable
+  properties: GeoCircleViewLayer
+  stylingConfig: Partial<Config>
+}
+
+const formatRowInfo = (
+  properties: GeoCircleViewLayer,
+  table: GeoTable,
+  index
+) => {
+  const result = []
+  const timeValue = table.getTimeString(index)
+  if (timeValue)
+    result.push({
+      key: '_time',
+      name: 'Time',
+      type: 'string',
+      values: [timeValue],
+    })
+  const {colorField, radiusField} = properties
+  const radiusValue = table.getValue(index, radiusField)
+  const {radiusDimension} = properties
+  const radiusInfo = formatValue(
+    radiusField,
+    'Radius',
+    radiusValue,
+    radiusDimension
+  )
+  if (radiusInfo) result.push(radiusInfo)
```

Comment:
Please always use braces for if statements. We will be updating the eslint rules to reflect this.